### DISCUSSION TITLE
Comments - add notification option to let users of a community know that a Pub has received new comments

### DISCUSSION THREAD

**Discussion by tosteiner** [knowledgefutures/pubpub#2890] [Ideas]
We have been using PubPub for quite some time now, and really love how things have been developing over time!

Now, with some of our Pubs and collections, I just recently found out retrospectively that we had received comments quite a while ago that we would've loved to respond to at that time, but failed to do so simply because we didn't know that these had been posted ...

So, I was thinking, one potential new feature would make the lives of community admins easier would be a **notification option for comments** received on a community's Pubs, so that key people managing the community are being reminded about which of the Pubs in their community would benefit from interaction. 

Several notification options come to mind, e.g. to receive an email, or display new comments in a dedicated backend section (thinking something like the  [WordPress comments section](https://wordpress.org/documentation/article/comments-in-wordpress/)). Responses to comments could also pop up in a given user's personal account space, to let them know that somebody has engaged with their comment ... just some thoughts ...

**Comment by gabestein** [comment]
Hello! We actually have both email alerts available and a dedicated notifications "tray" for comments! Here's where the tray is:

<img width="486" alt="Screenshot 2023-11-01 at 12 47 43" src="https://github.com/pubpub/pubpub/assets/639110/93bb2623-4318-474a-8f37-77e056c4563d">

These settings can also be found in the user's privacy settings (https://pubpub.org/legal/settings).

To receive emails, all you need to do is turn on both the notifications and discussion thread emails settings.

At present, the emails are a bit noisy, but it is easy to unsubscribe to any given thread/Pub either in the tray or by visiting the Pub itself and clicking the bell icon in the comments section.

**Comment by tosteiner** [comment]
oooh, brilliant - thanks so much for the pointer, Gabe! 🙏🏽
